src/mbgl/style/conversion/tileset.cpp

```diff
@@ -65,6 +70,28 @@ optional<Tileset> Converter<Tileset>::operator()(const Convertible& value, Error
         result.attribution = std::move(*attribution);
     }
 
+    auto boundsValue = objectMember(value, "bounds");
+    if (boundsValue) {
+        if (!isArray(*boundsValue) || arrayLength(*boundsValue) != 4) {
+            error = { "bounds must be an array with left, bottom, top, and right values" };
+            return {};
+        }
+        optional<double> left = toDouble(arrayMember(*boundsValue, 0));
+        optional<double> bottom = toDouble(arrayMember(*boundsValue, 1));
+        optional<double> right = toDouble(arrayMember(*boundsValue, 2));
+        optional<double> top = toDouble(arrayMember(*boundsValue, 3));
+
+        if (!left || !right || !bottom || !top) {
+            error = { "bounds array must contain numeric longitude and latitude values" };
+            return {};
+        }
+        if (!validateLatitude(*bottom) || !validateLatitude(*top)){
+            error = { "bounds latitude values must be between -90 and 90" };
+            return {};
+        }
```

Comment:
we should also verify that bottom <= top (and left <= right?)

Reply:
@kkaefer, as part of fixing left <= right, should we also clamp the longitude to [-180, 180] as discussed [here](https://github.com/mapbox/mapbox-gl-native/pull/10701#discussion_r157892072)?